asset/index.html

```diff
@@ -0,0 +1,404 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+    <meta http-equiv="x-ua-compatible" content="ie=edge"/>
+    <title>Plexus</title>
+    <!-- MDB icon -->
+    <link rel="icon" href="static/img/mdb-favicon.ico" type="image/x-icon"/>
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css"/>
+    <!-- Google Fonts Roboto -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"/>
+    <!-- MDB -->
+    <link rel="stylesheet" href="static/css/mdb.min.css"/>
```

Comment:
The issue says, that the files from mdb5 should be included via a CDN, so there should be no need for them to be included into this project. 

> Let's include a basic CSS from the CDN of https://mdbootstrap.com/docs/standard/getting-started/installation/